#FAQ
===

Frequently asked questions about using Github scripts or working on Beocat

##Using Github

See https://github.com/i5K-KINBRE-script-share/FAQ/blob/master/UsingGithubScripts.md

##Getting started and logging in to Beocat or an SGE cluster

See https://github.com/i5K-KINBRE-script-share/FAQ/blob/master/UsingBeocat.md

##Uploading, downloading and editing files on Beocat or an SGE cluster

For general instructions see https://github.com/i5K-KINBRE-script-share/FAQ/blob/master/BeocatEditingTransferingFiles.md. For instructions on getting sequence data from your sequencing facillity, NCBI's FTP, or the Short Read Archive (SRA) see https://github.com/i5K-KINBRE-script-share/FAQ/blob/master/GetRawData.md.

##Trouble logging onto Beocat?

If you are having trouble logging in on campus make sure you are connected the ksuwireless rather than guest. To login to Beocat from off campus you will need to VPN into the network. See the following link for instructions.

http://www.k-state.edu/its/security/vpn/


##Finding NGS workflows and tutorials for Beocat

See http://i5k-kinbre-script-share.github.io/transcriptome-and-genome-assembly/

##Helpful Beocat or an SGE cluster commands

See: https://github.com/i5K-KINBRE-script-share/FAQ/blob/master/BeocatHelpfulCommands.md

##Report a new issue about these scripts

If your question was not answered here see the link below

See: https://github.com/i5K-KINBRE-script-share/FAQ/blob/master/ReportIssues.md

##Using Trinity on Beocat:

Dong Hun Kim has made an excellent write up of how to install [Trinity on Beocat](https://github.com/kp5091/beocat-Trinity)



